demo-projects/meetup/site/pages/index.js

```diff
@@ -45,37 +38,41 @@ export default function Home() {
         <a>Sign In</a>
       </Link>
       <h2>Upcoming Events</h2>
-      <Query query={GET_ALL_EVENTS} variables={{ date: date.toISOString() }}>
+      <Query query={GET_EVENTS_AND_SPONSORS}>
         {({ data, loading, error }) => {
           if (loading) return <p>loading...</p>;
           if (error) {
             console.log(error);
             return <p>Error!</p>;
           }
+          const { allEvents, allSponsors } = data;
+          const now = Date.now();
 
-          return (
-            <ul>
-              {data.allEvents.map(event => (
-                <EventItem key={event.id} {...event} />
-              ))}
-            </ul>
-          );
-        }}
-      </Query>
-      <h2>Sponsors</h2>
-      <Query query={GET_ALL_SPONSORS}>
-        {({ data, loading, error }) => {
-          if (loading) return <p>loading...</p>;
-          if (error) {
-            console.log(error);
-            return <p>Error!</p>;
+          let eventItem = null;
```

Comment:
I'd like to separate the concepts of picking the correct event to display and rendering JSX in this block, but happy to have that happen in a separate PR